src/Tribe/Rewrite.php

```diff
@@ -632,32 +632,47 @@ public function filter_rewrite_parse_query_vars( array $query_vars = [], array $
 	 *               an entry..
 	 */
 	protected function get_option_controlled_slug_entry( array $localized_matchers, $default_slug, $option_name ) {
-		$using_default_archive_slug = $default_slug === tribe_get_option( $option_name, $default_slug );
+		$current_slug       = tribe_get_option( $option_name, $default_slug );
+		$using_default_slug = $default_slug === $current_slug;
 
-		$filter = static function ( $matcher ) use ( $default_slug )
-		{
+		$filter = static function ( $matcher ) use ( $default_slug ) {
 			return isset( $matcher['query_var'] )
-			       && 'post_type' === $matcher['query_var']
-			       && isset( $matcher['localized_slugs'] )
-			       && is_array( $matcher['localized_slugs'] )
-			       && in_array( $default_slug, $matcher['localized_slugs'], true );
+				   && 'post_type' === $matcher['query_var']
+				   && isset( $matcher['localized_slugs'] )
```

Comment:
:warning: Found precision alignment of 3 spaces.

<sub>hash: 16fce8364f11cbcd3c4d08748c45fb6f</sub>